pkg/controller/cincinnati/new.go

```diff
@@ -196,6 +199,32 @@ func (k *kubeResources) newPolicyEngineService(instance *cv1beta1.Cincinnati) *c
 	}
 }
 
+func (k *kubeResources) newPolicyEngineRoute(instance *cv1beta1.Cincinnati) *routev1.Route {
+	name := namePolicyEngineRoute(instance)
+	return &routev1.Route{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: instance.Namespace,
+			Labels: map[string]string{
+				"app": name,
```

Comment:
For a `Cincinnati` with name `foo`, the route name would be `foo-policy-engine-route`, which is not exactly the "app" name. Perhaps this would be better if the value were the Deployment's name, which would then match the label on the Deployment.

Reply:
Done.